### PR TITLE
Basic PvpArena contract

### PR DESCRIPTION
**Pull request by RafaelVidaurre**
* Creates a skeleton for the PvpArena contract. It contains mostly the shape of the contracts without much implementation, just to get ourselves something to look at
* Fixes tests migrations to have contracts in their latest state
* Adds tests directory to eslint, let's be careful with this though, might be breaking the cryptoblades team's pipeline
* Adds some minimalistic tests just to see if we got them working properly
* Renames test migrations folder to be a bit more explicit in regards to purpose

### CODE REVIEW

File: contracts/PvpArena.sol [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    /// @notice enter the arena with a character and a weapon
+    function enterArena(uint256 characterID, uint256 weaponID) public {
+        // TODOS:
+        // - [ ] check if character is not in the arena
+        // - [ ] check if weapon is not in the arena
+        // - [x] calculate wager cost
+        // - [ ] check if character is sender's
+        uint256 entryCost = getEntryCost(characterID);
+
+        skillToken.transferFrom(msg.sender, address(this), entryCost);
```
Comment (rahulthakkar796) [inline contracts/PvpArena.sol:65]: `transferFrom` method should be used at the end of the method body, it's a standard practice in solidity.

Comment (RafaelVidaurre) [reply contracts/PvpArena.sol:65]: Fixing! I'm guessing this is because if we have a revert the money is basically lost, right?

Comment (rahulthakkar796) [reply contracts/PvpArena.sol]: No, But it's a practice to avoid attacks such as reentrancy attacks.

Comment (RafaelVidaurre) [reply contracts/PvpArena.sol]: Oh right, that makes sense!

**Review by rahulthakkar796 — APPROVED** [review]

**Pull request merged**